### PR TITLE
Fix ticket field data source marshalling

### PR DESCRIPTION
**Pull request by jlsalmon**

### CODE REVIEW

File: zendesk/data_source_zendesk_ticket_field_test.go [hunk 1]
```diff
@@ -13,7 +14,7 @@ import (
 
 const systemFieldConfig = `
 data "zendesk_ticket_field" "assignee" {
```
Comment (tamccall) [inline zendesk/data_source_zendesk_ticket_field_test.go]: we should probably fix the example ticket fields as well. https://github.com/nukosuke/terraform-provider-zendesk/blob/master/examples/ticket_fields.tf#L58-L81

File: zendesk/resource_zendesk_organization.go [hunk 2]
```diff
@@ -112,7 +112,7 @@ func unmarshalOrganization(d identifiableGetterSetter) (client.Organization, err
 	}
 
 	if v, ok := d.GetOk("group_id"); ok {
-		org.GroupID = v.(int64)
+		org.GroupID = int64(v.(int))
```
Comment (tamccall) [inline zendesk/resource_zendesk_organization.go]: Does all of this organization / target related code belong in this pr? should we split those changes off into their own?

File: zendesk/data_source_zendesk_ticket_field_test.go (hunk 1 above)
Comment (tamccall) [reply zendesk/data_source_zendesk_ticket_field_test.go]: i don't know if it needs to be @nukosuke or myself but someone should probably update the wiki as well. https://github.com/nukosuke/terraform-provider-zendesk/wiki/Data-Source:-zendesk_ticket_field

Comment (jlsalmon) [reply zendesk/data_source_zendesk_ticket_field_test.go]: I already updated the examples in the `terraform-v0.12` branch (which this PR targets). I would update the wiki as well, but I'm not sure I have permission.

File: zendesk/resource_zendesk_organization.go (hunk 2 above)
Comment (jlsalmon) [reply zendesk/resource_zendesk_organization.go]: You're right, they don't really belong here - I just lazily lumped them into this PR. They aren't essential, just a bit of cleanup. I'll remove them from here.

File: zendesk/data_source_zendesk_ticket_field_test.go [hunk 3]
```diff
@@ -36,12 +37,14 @@ func TestTicketFieldDataSourceRead(t *testing.T) {
...
 	c.EXPECT().GetTicketFields(gomock.Any()).Return([]zendesk.TicketField{out}, zendesk.Page{}, nil)
+	c.EXPECT().GetTicketField(gomock.Any(), gomock.Any()).Return(out, nil)
+
 	err = readTicketFieldDataSource(m, c)
 	if err != nil {
 		t.Fatalf("Read system field returned an error. %v", err)
 	}
-
-	if v, ok := m.GetOk("id"); !ok || v.(int64) != out.ID {
+
```
Comment (nukosuke) [inline zendesk/data_source_zendesk_ticket_field_test.go]: please `go fmt`

Comment (jlsalmon) [reply zendesk/data_source_zendesk_ticket_field_test.go]: done!

**Pull request merged**